WordPress/Classes/ViewRelated/Domains/RegisterDomain.storyboard

```diff
@@ -0,0 +1,77 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
I'm not sure if this is actually finished, but just to point out some UI issues on this view:

![details](https://user-images.githubusercontent.com/1816888/43098275-ee182c56-8e7b-11e8-91e1-c5d5ae30d229.png)

- The back button text should be `Back` (not `Register domain`)
- The `Privacy Protection` header text overlaps the checkmark cells when scrolling.
- There is a gap between the checkmark cells and the `Domain Contact Information` header.

Reply:
you are right this is not finished yet, but thanks for putting effort on checking!